otter/cloud_client/__init__.py

```diff
@@ -501,6 +502,15 @@ class CLBNodeLimitError(Exception):
     """
 
 
+@attr.s
+class CLBPartialNodesRemoved(Exception):
+    """
+    Exception raised when some of the nodes are removed.
```

Comment:
I'm not sure if this is just my fault, but I ended up doing a bit of a double-take on this error object later on.  I think it would make more sense to say that the exception is raised when *only* some of the requested nodes are removed.